pybsn/test/test_switch.py

```diff
@@ -0,0 +1,56 @@
+import os
+from betamax import Betamax
+from requests import Session
+from nose import with_setup
+from ..api import Api
+
+with Betamax.configure() as config:
+    config.cassette_library_dir = 'fixtures/cassettes/switch'
+
+
+def test_get_switches():
+	api = Api(os.environ['PYBSN_HOST'], os.environ['PYBSN_USER'], os.environ['PYBSN_PASS'])
```

Comment:
Yeah. And Ideally make a common setup helper that we can use across all of our tests..